controls/pcidss_4.yml

```diff
@@ -1556,6 +1556,7 @@ controls:
       rules:
         - ensure_redhat_gpgkey_installed
         - ensure_suse_gpgkey_installed
+        - ensure_almalinux_gpgkey_installed
```

Comment:
IMO it would've been easier to use jinja here like:
```
{{% if "almalinux" in product %}}
        - ensure_almalinux_gpgkey_installed
{{% endif %}}
```
But this also works for sure

Reply:
@Xeicker yes as my first real dive into CaC i noticed a few things like that that seem like they need fixing globally as they probably started off as a single item list and then got copy'n'pasted as each new product got added.

of course we'd need a jinja block for each product/key, not  just this new one.

as you say it works for this release, but maybe we can add a backlog item to go over all of the profiles (i also mentioned the oracle vs rhel difference regarding gpgkeys in a comment above).

Reply:
Could this be the problem with the failing CI test, although I don't understand why the SUSE key test wouldn't have the same issue with the RHEL product

Reply:
That test can be ignored actually, what happens is that the scripts detect what has change, one change is this new rule creation. So the CI tries to test that rule on all platforms that has this test enabled. It fails because it tries to test your new rule in RHEL9 but that rule is not available for that product.

Nothing to do on your side.

Reply:
Thanks for the explanation!